net/FlatBuffers/Accessor.cs

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2014 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace FlatBuffers
+{
+    /// <summary>
+    /// This is the base for both structs and tables.
+    /// </summary>
+    public interface Accessor
+    {
+        void __init(int _i, ByteBuffer _bb);
```

Comment:
Since this is in implementation (they are public, but not intended for the users of the API), lets do that in a followup commit, since this would cause yet more differentiation with the Java implementation. I think it would be good to split them both up, then we can do further cleanup on C# much more easily.